src/showcase/components/button/BorderGlowButton.tsx

```diff
@@ -22,16 +21,9 @@ const BorderGlowButton = () => {
   }, [])
 
   return (
-    <motion.button
-      className="relative overflow-hidden rounded-lg bg-[#e5e7eb]"
+    <button
+      className="relative overflow-hidden rounded-lg bg-[#e5e7eb] transform active:scale-95"
```

Comment:
Can you swap `transform active:scale-95` for `transition-transform ease-in-out active:scale-90`?

This adds a transition that is similar to what it was with Framer Motion (minus the springiness).

Reply:
Okay think u I will do it .